Master Project/Assets/Scenes/Stirring/ScoreKeeperScript.cs

```diff
@@ -0,0 +1,38 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using System;
+
+public class ScoreKeeperScript : MonoBehaviour {
+
+
+    public float score { get; private set; }
+
+    private readonly Guid _NESSIE_GUID = new Guid("{060F70EA-8A92-4117-AB65-75DE3458E407}");
+
+    // Use this for initialization
+    void Start () {
+
+    }
+	
+	// Update is called once per frame
+	void Update () {
```

Comment:
This method is both empty and indented differently than everything else (this uses tabs, the rest uses spaces) - can it be removed?